### ISSUE TITLE
Write a (fast) postgresql backend

### ISSUE THREAD

**Issue by janLo** [agdsn/traffic-service-server#6]
Based on the interface from issue #4 there shouls be a backend using a Postgresql database for the fnal usecase.

**Comment by vale981** [comment]
Is this still a matter?
I would be keen to get hacking with agdsn and this looks interesting and doable for me.

Cheers,
Valentin

**Comment by mmarx** [comment]
No, this project is obsolete. Check out https://github.com/agdsn/pycroft/issues for an active project that could use your help, or just come by one of our meetings (Tuesdays at 19:30, with the focus rotating between network, computing infrastructure, and services).

**Comment by vale981** [comment]
Alright. I'll come as soon as I move to Dresden in August.
Also, I'll have a look at pycroft.

**Comment by mmarx** [comment]
Nice, looking forward to meeting you. :)

**Comment by vale981** [comment]
Me too :)

**Comment by janLo** [comment]
This project is somehow stale. I don't know if its really obsolete.

**Comment by vale981** [comment]
Well, I like C++.
So if you want to revive it :P...

**Comment by janLo** [comment]
The Idea was to have a microservice that delivers data from a database that
is filled by flow data collector. I'm too long out of the group to be able
to say if such a thing is still needed.

**Comment by lukasjuhrich** [comment]
@vale981 in the meantime, feel free to join us on our public XMPP/Jabber Chatroom over at `agdsn@conference.agdsn.de`. The `agdsn` channel is quite generic and not bound to any topic, so it's perfect for asking questions and small talk.

(If you are not familiar with Jabber, see https://www.jabber.org/. A solid client is e.g. [pidgin](https://pidgin.im/).)

**Comment by vale981** [comment]
Is you jabber room down? I can't connect to it through any client...

**Comment by mmarx** [comment]
No, it is working quite fine. Which client(s) are you using, and from which server are you trying to connect?

**Comment by vale981** [comment]
Note: 4ept.net doesn't work
